tests/tests/smoke_tests.robot

```diff
@@ -24,9 +21,8 @@ Extract test file
     [Arguments]    ${test_file}
     # TODO: Move some steps to TEST SETUP or SUITE SETUP
     ${downloaded_file_path}    Set Variable    ${DOWNLOAD_PATH}${test_file}[archive_name].zip
```

Comment:
as above

Reply:
Agreed with Krzysztof that 'Set variable' topic will be handle by separate PR.